### PR TITLE
Update translations for base_company and list_messages templates

### PR DESCRIPTION
**Pull request by seppeljordan**
This change deals exclusively with translations of user facing strings. All other changes are only formatting changes to the source file without any runtime implications.

Plan-ID: 1906a785-a23b-44b3-a599-96ee43dd388c

### CODE REVIEW

**Comment by seppeljordan** [comment]
You are right. I mixed up the translation task.

File: arbeitszeit_flask/templates/base_company.html [hunk 1]
```diff
@@ -63,46 +63,46 @@
...
-                Konten
+                {{ gettext("Accounts") }}
               </a>
               <a href="{{ url_for('main_company.transfer_to_worker') }}" class="navbar-item">
-                Bezahlen Mitarbeiter
+                {{ gettext("Pay workers") }}
               </a>
               <a href="{{ url_for('main_company.transfer_to_company') }}" class="navbar-item">
-                Bezahlen Produktionsmittel
+                {{ gettext("Pay means") }}
```
Comment (knit-bee) [inline arbeitszeit_flask/templates/base_company.html:77]: For "Produktionsmittel" maybe "means of prodcution" or "resources" would be better

File: arbeitszeit_flask/templates/base_company.html [hunk 2]
```diff
@@ -114,20 +114,20 @@
           <div class="buttons">
             {% if not current_user.is_authenticated %}
             <a class="button" href="{{ url_for('auth.login_company') }}">
-              Anmelden
+              {{ gettext("Sign in") }}
             </a>
             <a class="button" href="{{ url_for('auth.signup_company') }}">
-              Registrieren
+              {{ gettext("Register") }}
```
Comment (knit-bee) [inline arbeitszeit_flask/templates/base_company.html:120]: Isn't this normally "sign up" in Internet language?

**Review by knit-bee — COMMENTED** [review]
I would translate some of the strings differently but I think it is okay like this anyhow.

**Comment by seppeljordan** [comment]
@knit-bee I implemented the suggestions.

File: arbeitszeit_flask/templates/base_company.html (hunk 1 above)
Comment (sloschert) [reply arbeitszeit_flask/templates/base_company.html]: > @knit-bee I implemented the suggestions.

Don't want to be petty, but would "Pay means of production" be ok as well?

**Review by sloschert — APPROVED** [review]
Thanks, I had a small comment, but anyway it's approved. 
Certificates transferred.

**Pull request merged**